Plugin/ClearQuote.php

```diff
@@ -25,14 +28,69 @@
  */
 class ClearQuote
 {
+    /**
+     * @var CartHelper
+     */
+    private $cartHelper;
+
+    /**
+     * @var \Magento\Quote\Model\Quote|null
+     */
+    private $quote_to_restore;
+
+    /**
+     * @param CartHelper $cartHelper
+     */
+    public function __construct(
+        CartHelper $cartHelper
+    )
+    {
+        $this->cartHelper = $cartHelper;
+    }
+
+    /**
+     * @param CheckoutSession $subject
+     * @return null Return null because method clearQuote have no arguments
+     * @throws \Exception
+     */
+    public function beforeClearQuote(CheckoutSession $subject)
+    {
+        // We don't want to clear quote
+        // in Product page checkout (PPC) flow
+        $this->quote_to_restore = null;
+        $current_quote_id = $subject->getQuote()->getId();
+        $order_quote_id = $subject->getLastSuccessQuoteId();
```

Comment:
Can you use a proper variable naming?
https://devdocs.magento.com/guides/v2.3/coding-standards/code-standard-php.html
It is related to the
- `$this->quote_to_restore `
- `$current_quote_id`
- `$order_quote_id`

Reply:
Done, thank you